tachyon/math/finite_fields/fp12.h

```diff
@@ -282,8 +279,7 @@ class Fp12 final : public QuadraticExtensionField<Fp12<Config>> {
     this->c1_ -= b;
     // c0 = ((α₃β₃ + α₅β₄q) + (α₄β₃ + α₃β₄)x + (α₅β₃ + α₄β₄)x²)p +
     //      α₀β₀ + α₁β₀x + α₂β₀x², where p = y² and q = x³
-    this->c0_ = Config::MulByNonResidue(b);
-    this->c0_ += a;
+    this->c0_ = Config::MulByNonResidue(b) + a;
```

Comment:
I don't think this should be changed:
its equal to the case 
```c++
z = b * c + a
```
which should be treated as such:
```c++
z = b * c;
z += a;
```

 REF[0] for future similar scenarios